docs/reference/modules/terraform-aws-asg/asg-instance-refresh/asg-instance-refresh.md

```diff
@@ -73,6 +76,138 @@ The Terraform [instance_refresh](https://registry.terraform.io/providers/hashico
 *   Auto Scaling Groups support up to one active instance refresh at a time. When this resource is updated, any existing refresh is cancelled.
 *   Depending on health check settings and group size, an instance refresh may take a long time or fail. The Terraform `instance_refresh` resource does not wait for the instance refresh to complete.
 
+## Sample Usage
+
+<ModuleUsage>
+
+```hcl title="main.tf"
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY GRUNTWORK'S ASG-INSTANCE-REFRESH MODULE
+# ---------------------------------------------------------------------------------------------------------------------
+
+module "asg-instance-refresh" {
```

Comment:
NIT: use snake case for TF module names. E.g., Convert this to `asg_instance_refresh`.

Reply:
✅